### ISSUE TITLE
Support for mobile 

### ISSUE THREAD

**Issue by Jopp-gh** [dorisxx/Obsidian-simple-note-quiz#13]
Your plugin is unique and useful. I don't like Anki, or Anki integrations with Obsidian because too complicated 
Please make your nice plugin compatible for  mobile as well!👍

**Comment by beginner137** [comment]
Hi, I'll try to extend the plugin to mobile, thanks for the advice!

**Comment by Jopp-gh** [comment]
Sorry I forgot about, you just need to set the boolean in the manifest from false to true, that's all

**Comment by beginner137** [comment]
yeah that's like the last step, before that I need to test to make sure everything renders and functions well on mobile devices.
